examples/radial_gradient_filters.py

```diff
@@ -48,7 +48,7 @@
 # choose them according to your requirements.
 
 order = 20
-attenuation_coefficients = radial.set_attenuation_coefficients(order)
+attenuation_coefficients = radial._set_attenuation_coefficients(order)
```

Comment:
We do not want to encourage use of the private function, the rest of the example needs to be updated to use the new keywords.